Fotogjengen/hilfling-frontend
# Feature/endre/navbar v2

What is done
- Made a new navigation bar with hamburger menu
- made a "logg inn" component loading when "/logginn"
- changed the layout of front page to be more responsive
- removed the bugged carousel 
- changed all content to be inside a centered 90vw box

How to check
 - "npm start"
 - check if frontpage is working and resonsiv
 - check if navigation bar is working and responsive
 - check if hamburger menu appears when screen becomes smaller
 - check if navigation bar disappears when screen becomes smaler
 - check if hamburger menu works
 - check if every link works as before
 - check if design good
